src/atoms/Size/Size.web.js

```diff
@@ -1,3 +1,31 @@
-const Size = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Size = styled(
+  ({ minWidth, width, maxWidth, minHeight, height, maxHeight, children, ...props }) => {
+    if (React.Children.toArray(children).length !== 1) {
+      throw new Error('Expected a single child for Size component');
+    }
+
+    return React.cloneElement(children, props);
+  },
+)`
+  ${(props) => (props.width ? `width: ${props.width}` : '')};
```

Comment:
all the props here should be 
```js
${(props) => (props.width ? `width: ${makePxValue(props.width)}` : '')};
```
refer space.web.js for reference

I don't know how this got missed on native 🙊 
cc @shubhnik

Reply:
@kamleshchandnani I would fix it and raise a PR. I remember I just changed the `cloneElement` thing in `<Size />` but should have caught this mistake earlier 🙈

Reply:
this is done